.github/workflows/audit.yaml

```diff
@@ -0,0 +1,20 @@
+name: Dependency Audit
+on: workflow_dispatch
+# on:
```

Comment:
Nitpick: if we need to keep these comments in here for future reasons, let's add a comment explaining. Otherwise we can go ahead and remove it

Reply:
[391fb73](https://github.com/Exein-io/pulsar/pull/30/commits/391fb7395eae889274c0ddf49fc9d2bfd5410da8)